lambdas/GitHubAccessCodeGetter/lambda_function.py

```diff
@@ -0,0 +1,37 @@
+import os
+import requests
+import json
+
+client_id = os.environ['CLIENT_ID']
+client_secret = os.environ['CLIENT_SECRET']
+org_whitelist = os.environ['ORG_WHITELIST'].split(';')
+ignore_whitelist = os.environ['IGNORE_WHITELIST']
+
+def generate_resp(code, body):
```

Comment:
This occurs more than once, and could be used in even more places. Consider beefing up this function with kwargs and externalizing it.